Stratis.Bitcoin/FullNode.cs

```diff
@@ -1,286 +1,336 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Globalization;
-using System.Text;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Hosting;
+﻿using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Logging;
 using NBitcoin;
+using Stratis.Bitcoin.Base;
 using Stratis.Bitcoin.Builder;
+using Stratis.Bitcoin.Common.Hosting;
 using Stratis.Bitcoin.Configuration;
-using Stratis.Bitcoin.Connection;
 using Stratis.Bitcoin.Configuration.Logging;
-using Stratis.Bitcoin.Utilities;
-using System.Reflection;
-using Stratis.Bitcoin.Base;
-using Stratis.Bitcoin.Common;
-using Stratis.Bitcoin.Common.Hosting;
+using Stratis.Bitcoin.Connection;
 using Stratis.Bitcoin.Features.BlockStore;
 using Stratis.Bitcoin.Features.Consensus;
 using Stratis.Bitcoin.Features.Consensus.CoinViews;
 using Stratis.Bitcoin.Features.MemoryPool;
 using Stratis.Bitcoin.Features.Wallet;
+using Stratis.Bitcoin.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
 
 namespace Stratis.Bitcoin
 {
+    /// <summary>
+    /// Node providing all supported features of the blockchain and its network.
+    /// </summary>
+    public class FullNode : IFullNode
+    {
+        /// <summary>Instance logger.</summary>
```

Comment:
ah ah ok it means the logger for this instance, not an instance of  a Logger.
What about "this instance's logger"?
If not then leave it as it is, it's fine.